docs/modules/operation/pages/deep-dive/admin/webui/ui-preview.adoc

```diff
@@ -2,15 +2,17 @@
 [[ui-preview]]
 = UI Preview
```

Comment:
Just a FYI: this whole section will be deleted in H31, as they are reintegrating the UI Preview screens into the main UI. However, best to keep this here in case there is a change of plans.